src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -153,6 +153,7 @@ class InterfacesUpdater(MIBUpdater):
     def __init__(self):
         super().__init__()
         self.db_conn = mibs.init_db()
+        self.multi_db_conn = mibs.init_multi_db() 
```

Comment:
>init_multi_db [](start = 34, length = 13)

I guess you need to init_multi_db before init_db. The init_multi_db will trigger a database_config initialization for all the namespaces including host namespace.

Reply:
fixed as per comment.